src/internal/components/cartesian-chart/chart-container.tsx

```diff
@@ -28,12 +29,16 @@ export const CartesianChartContainer = forwardRef(
       bottomAxisLabel,
       chartPlot,
       popover,
+      hasFilters,
     }: CartesianChartContainerProps,
     ref: React.Ref<HTMLDivElement>
   ) => {
     if (fitHeight) {
       return (
-        <div className={clsx(styles['chart-container'], styles['fit-height'])} ref={ref}>
+        <div
+          className={clsx(styles['chart-container'], styles['fit-height'], hasFilters && styles['has-filters'])}
```

Comment:
Wouldn't it be simpler to apply `has-filters` class to the `chart-container-outer` rather than using the complex css below?

Reply:
Good point, updated